wiki/Rules/Explicit_Content/de.md

```diff
@@ -0,0 +1,28 @@
+# Expliziter Inhalt
+
+![Screenshot einer Beatmap, die expliziten Inhalt enthält](img/explicit-tag-DE.jpg "Ein Beispiel einer Beatmap, die mit dem Tag 'explizit' markiert wurde.")
+
+Mapper können den Inhalt ihrer Beatmap als *explizit* zu deklarieren, indem sie die Checkbox `Explicit Content` ankreuzen, welche im [Genre- und Sprach-Popup-Menü](/wiki/Beatmap/Genre_and_language) auf dem Webeintrag der jeweiligen Beatmap zu finden ist. An diese Funktion sind einige Erwartungen und Regeln bezüglich ihrer Verwendung geknüpft.
```

Comment:
```suggestion
Mapper können den Inhalt ihrer Beatmap als *explizit* deklarieren, indem sie die Checkbox `Explicit Content` ankreuzen, welche im [Genre- und Sprach-Popup-Menü](/wiki/Beatmap/Genre_and_language) auf dem Webeintrag der jeweiligen Beatmap zu finden ist. An diese Funktion sind einige Erwartungen und Regeln bezüglich ihrer Verwendung geknüpft.
```

Reply:
Oh btw, I just notice that I originally wanted to check if the text beneath the checkbox is translated or not. Unfortunately, I'm not knowledgeable at all in the Beatmap Submission System, so I'm not sure where to look at for this.

Reply:
It is indeed translated, [just checked](https://pek.li/6fdj8f.png). Updated suggestion.

Reply:
Thanks for checking!

Reply:
fyi, you can see these settings on your own uploaded maps, when you click the edit button that appears near the genre and tags on hover.

Reply:
oh okay, nice to know! I once mapped a song by the pegboard nerds but I haven't uploaded it yet, so I only have very basic experiences with mapping.